seqr/views/apis/data_manager_api.py

```diff
@@ -107,7 +107,7 @@ def upload_qc_pipeline_output(request):
 
     info_message = 'Parsed {} {} samples'.format(
         len(json_records), 'SV' if dataset_type == Sample.DATASET_TYPE_SV_CALLS else data_type)
-    logger.info(info_message)
+    logger.info(info_message, extra={'user': request.user})
```

Comment:
The log here seems not necessary.

Reply:
its helpful to log as much detail as possible around big bulk updates like this one, so in case anything ever goes wrong we have more information to work with